### PR TITLE
Only need to sign CLA once for multiple orgs and repos configured wit…

### PR DESCRIPTION
**Pull request by MichaelTsengLZ**
UI:
screen shot 2017-07-20 at 1 15 59 am
screen shot 2017-07-19 at 11 14 30 pm
screen shot 2017-07-19 at 11 56 14 pm

### CODE REVIEW

File: src/client/templates/cla.html [hunk 1]
```diff
@@ -34,6 +34,9 @@ <h4 ng-show="signed">You have signed the CLA for {{params.user}}/{{params.repo}}
 						>
 					</customfield>
 				</form>
+				<p class="row" ng-if="linkedItem.sharedGist">
+					<em>* This CLA applies to multiple repos or orgs.<em>
```
Comment (jeffmcaffer) [inline src/client/templates/cla.html]: Nice.  Might want this to be more prominent.  For example, higher up and perhaps a bit larger font...

File: src/server/api/cla.js [hunk 2]
```diff
@@ -339,6 +360,42 @@ module.exports = {
...
     },
 
+    validateSharedGistItems: function (req) {
+        var self = this;
+        getLinkedItemsWithSharedGist(req.args.gist, function (error, sharedItems) {
+            if (error) {
+                log.error(error);
+            }
+            var items = [];
+            if (sharedItems.repos && sharedItems.repos.length) {
```
Comment (jeffmcaffer) [inline src/server/api/cla.js]: these if's (lines 369-375) could be simplified to on line
   var items = (sharedItems.repos || []).concat(sharedItems.orgs || [])

Comment (MichaelTsengLZ) [inline src/server/api/cla.js]: Nice and clean.

**Comment by MichaelTsengLZ** [comment]
The order of the questions is first choosing a repo/org and then choosing CLA. William and I don't like that and changed that in another PR.

**Comment by MichaelTsengLZ** [comment]
How about this: @jeffmcaffer 
screen shot 2017-07-20 at 9 36 23 am

**Comment by jeffmcaffer** [comment]
The UI order change feels gratuitous. would be interesting to talk to you and @willbar.

Consider removing the '*' on the info line.  * tends to indicate required.

**Pull request merged**